README.md

```diff
@@ -70,6 +71,7 @@ All parameters are optional.
 - [`classValidation`]: (default: `"false"`) - Add validation decorators from `class-validator`. Not compatible with `noDependencies = "true"` and `outputType = "interface"`.
 - [`noDependencies`]: (default: `"false"`) - Any imports and decorators that are specific to NestJS and Prisma are omitted, such that there are no references to external dependencies. This is useful if you want to generate appropriate DTOs for the frontend.
 - [`outputType`]: (default: `"class"`) - Output the DTOs as `class` or as `interface`. `interface` should only be used to generate DTOs for the frontend.
+- [`prettier`]: (default: `"false"`) - Should the output files should be stylized with Prettier?
```

Comment:
Twice `shoud` in the sentence.
Or maybe just a simple phrase without question: `Stylize output files with prettier.`